sdk/python/setup.py

```diff
@@ -63,6 +63,7 @@
     "tqdm==4.*",
     "fastapi>=0.68.0",
     "uvicorn[standard]>=0.14.0",
+    "dask==2021.9.1",
```

Comment:
I think hardcoding this dependency is too restrictive. Is there any reason to pin it?